Pair_Up/ClearPiece.h

```diff
@@ -0,0 +1,6 @@
+#ifndef CLEARPIECE_H
+#define CLEARPIECE_H
+#include "Map.h"
+Map clearPiece(Map temporary);//����Map�࣬������Ҫ��ȥ��Map��
```

Comment:
Because tgf said I'll return the object, so I don't think I need to send the parameter by reference.
Use the object as the parameter, maybe I need it in the future.
@Wafer-Li